runs/comparisons/rf.py

```diff
@@ -39,13 +44,14 @@
         X_train=X_train,
         y_train=y_train,
         random_state=random_state,
-        n_calls=20,
+        n_calls=1000,
         verbose=10,
         n_jobs_cv=4,
     )
 
     # Create the base experiment, using some default tuner
-    experiment = Experiment(name='Random Forest', tuner=default_tuner, verbose=10)
+    experiment_name = f'Random Forest {job_id} {problem}'
+    experiment = Experiment(name=experiment_name, tuner=default_tuner, verbose=10)
```

Comment:
What is the default_tuner? Shouldn't this be one of the ones we defined above?

Reply:
default_tuner is SkoptTuner. If we don't need that one I can remove it

Reply:
We should use optuna here

Reply:
SkoptTuner removed